### PR TITLE
Create verification-<Rudreshdabre12>.html

### PR DESCRIPTION
**Pull request by Rudreshdabre12**
Email-Verification_template

### CODE REVIEW

**Comment by Rudreshdabre12** [comment]
email verification template

File: src/middlewares/fileUpload.middleware.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    try {
+        const decoded = jwt.verify(token, secretKey);
+        req.userId = decoded.userId;
+        next();
+    } catch (error) {
+        return sendErrorResponse(res, 401, 'Unauthorized', 'Invalid or expired token');
+    }
+}
+
+function sendErrorResponse(res, statusCode, error, message) {
```
Comment (automatefreely) [inline src/middlewares/fileUpload.middleware.js:25]: this function is available in utils folder

File: src/middlewares/fileUpload.middleware.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+const jwt = require('jsonwebtoken');
+
+const secretKey = 'yourSecretKey'; 
+
+function generateToken(userId) {
```
Comment (automatefreely) [inline src/middlewares/fileUpload.middleware.js:5]: This function is already in user model

File: src/middlewares/fileUpload.middleware.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+const jwt = require('jsonwebtoken');
+
+const secretKey = 'yourSecretKey';
```
Comment (automatefreely) [inline src/middlewares/fileUpload.middleware.js:3]: Take it from .env

**Review by automatefreely — COMMENTED** [review]
> i have currently made a PR on middleware and email verification template

1. Don't make single PR of two issue, else you will not get point for both the issue. Also not a good practice.
2. Authorization middleware should be in auth.middleware.js

**Pull request closed without merging**